src/bpmn-visualization.ts

```diff
@@ -27,6 +27,7 @@ export { StyleConfigurator };
 export { ShapeUtil };
 // end of 'TO DO remove dedicated import/export'
 
+export * from './component/BpmnVisualization';
```

Comment:
:question: I am not sure we need this

Reply:
I was thinking to explicitly export BpmnVisualizationOptions

Reply:
The point here is BpmnVisualizationOptions is a TS interface so it doesn't exist in the transpiled Javascript. But this is going probably interesting for TypeScript integration.
So I will change the export to what you propose.